contracts/HubPool.sol

```diff
@@ -47,10 +43,22 @@ contract HubPool is Testable, Lockable, MultiCaller, Ownable {
 
     RefundRequest public refundRequest;
 
-    // Whitelist of origin token to destination token routings to be used by off-chain agents.
+    // Whitelist of origin token to destination token routings to be used by off-chain agents. The notion of a route
+    // does not need to include L1; it can store L2->L2 routes i.e USDC on Arbitrum -> USDC on Optimism as a "route".
     mapping(address => mapping(uint256 => address)) public whitelistedRoutes;
 
-    mapping(address => LPToken) public lpTokens; // Mapping of L1TokenAddress to the associated LPToken.
+    // Mapping of L1TokenAddress to the associated Pool information.
+    struct PooledToken {
+        address lpToken;
+        bool isEnabled;
+        uint256 liquidReserves;
+        int256 utilizedReserves;
+        uint256 undistributedLpFees;
+        uint32 lastLpFeeUpdate;
+        bool isWeth;
```

Comment:
Would we get gas savings (might not matter in the happy path) by moving the address, bools, and uint32 to all be next to one another?

Reply:
yup. will pack them more closely.

Reply:
I actually did this in a follow on PR. will leave it for that PR to make the change.